frontend/src/App.vue

```diff
@@ -55,12 +55,12 @@ export default {
         that.timetotal=(d-s)/1000;
         console.log(response.body);
```

Comment:
**[Required]** Should probably remove this, we shouldn't be logging anything to the console in a prod environment